Intellifire lights (#24574)
home-assistant/home-assistant.github.io: source/_integrations/intellifire.markdown

- **Thermostat** - This entity will be present if your unit has thermostatic control.

### Lights

> If the unit is equipped with lights you will have a light entity.

Reviewer: ```suggestion
The integration provides a light entity if the unit is equipped with lights.
```

- **Thermostat** - This entity will be present if your unit has thermostatic control.

> ### Lights

Reviewer: ```suggestion
### Light
```
